controllers/openshift_controller.go

```diff
@@ -24,6 +24,9 @@ import (
 
 	"k8s.io/apimachinery/pkg/labels"
 
+	operatorv1 "github.com/openshift/api/operator/v1"
+	cov1helpers "github.com/openshift/library-go/pkg/operator/v1helpers"
+
```

Comment:
Is there any guidelines to use the following project to handle Conditions in OpenShift operators - https://github.com/openshift/custom-resource-status ?